src/main.cpp

```diff
@@ -46,14 +46,15 @@ void setup() {
   wifiToLoraQueue = new DataQueue<message_t>(DATA_QUEUE_LENGTH);
   loraToWifiQueue = new DataQueue<message_t>(DATA_QUEUE_LENGTH);
 
-  mesh = new LoraMesh(wifiToLoraQueue, loraToWifiQueue, new Router());
+  // mesh = new LoraMesh(wifiToLoraQueue, loraToWifiQueue, new Router());
 
-  if (shouldEnableGateway(GATEWAY_SSID, GATEWAY_SSID_PASSWORD)) {
-    wifi = new Gateway(GATEWAY_SSID, GATEWAY_SSID_PASSWORD, wifiToLoraQueue,
-                       loraToWifiQueue);
-  } else {
-    wifi = new AccessPoint(ACCESS_POINT_SSID, wifiToLoraQueue, loraToWifiQueue);
-  }
+  // if (shouldEnableGateway(GATEWAY_SSID, GATEWAY_SSID_PASSWORD)) {
+  wifi = new Gateway(GATEWAY_SSID, GATEWAY_SSID_PASSWORD, wifiToLoraQueue,
```

Comment:
There's no need to change these lines to enable gateway. Use the platformio.ini configurations to do that.

Reply:
We already removed this comments, and we are using the platformio ini. Thanks!